tests/test_arxiv.py

```diff
@@ -44,13 +44,13 @@ def test_downloader_match(tmp_config: TemporaryConfiguration) -> None:
     from papis.arxiv import Downloader, ARXIV_ABS_URL
 
     down = Downloader.match("arxiv.org/sdf")
-    assert isinstance(down, Downloader)
```

Comment:
Why is this not needed anymore? It should assert here.

Reply:
Misread the code flow as a multi-step process to find down, instead of testing its value against multiple `Downloader.match` calls.